assets/package.json

```diff
@@ -73,54 +67,32 @@
 		"use-memo-one": "^1.1.2"
 	},
 	"devDependencies": {
-		"@babel/core": "^7.14.8",
-		"@babel/plugin-transform-runtime": "^7.10.4",
-		"@babel/preset-react": "^7.10.4",
+		"@babel/preset-react": "^7.26.3",
+		"@storybook/addon-queryparams": "^6.2.9",
```

Comment:
Shouldn't this be moved to the storybook package?

Reply:
No, this one should be here because this is how node.js loads dependencies. We import this package in `.story.js` files located in the `assets/js` folder, so node.js looks up for this package in `assets/node_modules` or `node_modules` folders.